client/src/App.js

```diff
@@ -15,6 +15,8 @@ import PICResult6 from "./views/PICResult6";
 import PICResult7 from "./views/PICResult7";
 import Account from "./views/Account";
 import Checklist from "./views/Checklist";
+import AboutUs from "./views/AboutUs";
```

Comment:
Are we still doing this page? Is this allocated to someone? Could the information on this page be transfer to an existing page?

Reply:
Comment actioned: deleted